packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -145,8 +147,10 @@ abstract class BindingBase {
                 break;
               case 'default':
               default:
+                value = 'android';
```

Comment:
We shouldn't be setting the default to android.

Reply:
passing back null to
_postExtensionChangedEvent would be better.

Reply:
as it indicates the state of no-override set.

Reply:
done.